pysb/bng.py

```diff
@@ -331,7 +331,7 @@ def _parse_netfile(model, lines):
             _parse_reaction(model, line)
         # fix up reactions whose reverse version we saw first
         for r in model.reactions_bidirectional:
-            if r['reverse']:
+            if all(r['reverse']):
```

Comment:
This is needed because `r['reverse']` is now a tuple. Whether to check `all()` or only the first entry in `r['reverse']` is a matter of preference of how to canonicalize which way round reversible rules are defined. Any opinions on this?